src/components/existing-request/existing-request-search.vue

```diff
@@ -111,6 +111,7 @@ import newReqModule from '@/store/new-request-module'
 // import ErrorModule from '@/modules/error' // *** TODO: delete this?
 // import { NameRequestI, SearchDataI, NrDataResponseT, NrDataT } from '@/interfaces' // *** TODO: delete this?
 
+// eslint-disable-next-line no-useless-escape
```

Comment:
Remove this override to see the error with the regex.

As long as the regex works, I'm not concerned about this. But does it work? 🤷‍♂️